enhancements/network/admin-network-policy.md

```diff
@@ -0,0 +1,322 @@
+        podSelector:
+          matchLabels:
+            conformance-house: slytherin
+  egress:
+  - name: "pass-all-egress-to-slytherin"
+    action: "Pass"
+    to:
+    - pods:
+        namespaces:
+          namespaceSelector:
+            matchLabels:
+              conformance-house: slytherin
+        podSelector:
+          matchLabels:
+            conformance-house: slytherin
+```
+
+The OVN-K controller will convert these into NBDB logical entities (showing only egress, but same will be done for ingress):
+
+```text
+_uuid               : b29f0a44-30b2-4857-985f-a2e899ea72cd
+acls                : [c02b4254-b1d0-48c1-a091-57ae6887bdc3, d4a487dd-3ac0-44d7-b07d-8d6eb0729567]
+external_ids        : {name=priority-50-example_25000}
+name                : a16480371363466183296
+ports               : []
+```
+```text
+_uuid               : 88a1905f-9733-479d-a23e-4e19aab9f95a
+addresses           : []
+external_ids        : {direction=ANPEgress, ip-family=v4, "k8s.ovn.org/id"="network-policy-controller:AdminNetworkPolicy:priority-50-example:ANPEgress:25000:v4", "k8s.ovn.org/name"=priority-50-example, "k8s.ovn.org/owner-controller"=network-policy-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy, priority="25000"}
+name                : a11410890234882820008
+```
+```text
+_uuid               : d4a487dd-3ac0-44d7-b07d-8d6eb0729567
+action              : pass
+direction           : from-lport
+external_ids        : {direction=ANPEgress, "k8s.ovn.org/id"="network-policy-controller:AdminNetworkPolicy:priority-50-example:ANPEgress:25000", "k8s.ovn.org/name"=priority-50-example, "k8s.ovn.org/owner-controller"=network-policy-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy, priority="25000"}
+label               : 0
+log                 : false
+match               : "((ip4.dst == $a11410890234882820008)) && (inport == @a16480371363466183296)"
+meter               : acl-logging
+name                : priority-50-example_ANPEgress_25000
+options             : {apply-after-lb="true"}
+priority            : 25000
+severity            : debug
+tier                : 1
+```
+
+#### Variation [optional]
+
+N/A
+
+### API Extensions
+
+None
+
+### Implementation Details/Notes/Constraints [optional]
+
+Covered in the above sections
+
+### Risks and Mitigations
+
+N/A
+
+### Drawbacks
+
+N/A
+
+## Design Details
+
+### Open Questions [optional]
+
+Aligning existing network policy V1 code with this new controller -
+we need to decide if we want to stick to existing event drivenness
+and add things into `oc` or if we want to go ahead with the new
+controller
+
+### Test Plan
+
+* e2e conformance tests will be added for ANP & BANP into upstream `network-policy-api` repo
+* Those e2e's will be brought down into OVN-Kubernetes CI and run as part of the presubmit jobs
+* unit testing will be added for both ANP & BANP, pod label and namespace label changes
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+Tech Preview: 4.14
```

Comment:
Sorry I keep getting confused here :) Why not start with "dev preview" ? 

Originally I thought for "tech preview" the API needed to make it to `beta` but now that's changed? 

It might be worth adding a quick blurb here noting why we chose to do either tech OR dev preview first?

Reply:
@abhat told me there is no such thing as "DevPreview" in OCP. It's either "TechPreview" or GA. Having said that, I am ok to keep this feature on TechPreview till 4.16. So TechPreview for 2 releases and then GA that should give us enough time to graduate the API.